package/src/Mutation/Mutation.test.ts

```diff
@@ -106,4 +106,10 @@ describe('Mutation', () => {
     expect(onAsyncError).toBeCalledWith('foo');
     expect(onDefaultError).not.toBeCalled();
   });
+
+  it('background равен null', () => {
```

Comment:
```suggestion
  it('Background равен null', () => {
```

Reply:
https://kaluga-astral.github.io/style-guide/docs/rules/tests/naming#%D1%82%D0%B5%D1%81%D1%82-%D0%BA%D0%B5%D0%B9%D1%81-%D0%B4%D0%BE%D0%BB%D0%B6%D0%B5%D0%BD-%D0%BD%D0%B0%D1%87%D0%B8%D0%BD%D0%B0%D1%82%D1%8C%D1%81%D1%8F-%D1%81-%D0%B1%D0%BE%D0%BB%D1%8C%D1%88%D0%BE%D0%B9-%D0%B1%D1%83%D0%BA%D0%B2%D1%8B